File: 1020 - Idade em Dias/main.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
let tot = parseInt(input);
let years = Math.floor(tot / 365);
let months = Math.floor(tot % 365 / 30);
let days = tot % 365 % 30;

console.log(years + ' ano(s)');
console.log(months + ' mes(es)');
console.log(days + ' dia(s)');

File: 1020 - Idade em Dias/main.ml
let tot = read_int()
let years = tot / 365
let months = tot mod 365 / 30
let days = tot mod 365 mod 30;;

Printf.printf "%d ano(s)\n" years;
Printf.printf "%d mes(es)\n" months;
Printf.printf "%d dia(s)\n" days
